src/Auth/Eloquent/RoleModel.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Statamic\Auth\Eloquent;
+
+use Illuminate\Database\Eloquent\Model as Eloquent;
+
+class RoleModel extends Eloquent
+{
+    protected $guarded = [];
+
+    protected $table = 'roles';
+
+    protected $casts = [
+        'permissions' => 'json',
+        'preferences' => 'json',
+        'created_at' => 'datetime',
+        'updated_at' => 'datetime',
+    ];
+
+    public function __construct(array $attributes = [])
+    {
+        parent::__construct($attributes);
+
+        if ($table = config('statamic.users.roles')) {
```

Comment:
```suggestion
        if ($table = config('statamic.users.tables.roles')) {
```

Reply:
@ryanmitchell I had to make this change to be able to log in but after this things seem to work smoothly.